docker-compose.yml

```diff
@@ -70,7 +70,7 @@ services:
       - elasticsearch
     build:
       context: .
-    command: -e local-compose -p DevelopmentSetupProcess
```

Comment:
When I change my local-compose file to point to localhost instead of the Docker container names, I do get a connection error when trying to connect to ElasticSearch. 

2024-08-05 17:38:35 urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPConnection object at 0x7f0d988c7e20>: Failed to establish a new connection: [Errno 111] Connection refused

I think this should still be `local-compose`.

Reply:
You were right to change the command back to using the local-compose file. Running the local.yaml file when running the ClusterProcess after running this docker compose file made the ClusterProcess work.